build/Dockerfile.client

```diff
@@ -1,17 +1,8 @@
-ARG GOLANG_IMAGE_VERSION=golang:1.20-alpine3.16
+ARG GOLANG_IMAGE_VERSION=golang:1.20-bookworm
```

Comment:
heh ur an infra guy too? yeah i made this change to have parity with the localnet since both were using debian. Alpine is great for prod, a bit tough on people in development.

In any case, what is ur action here and let's move it to a new issue as to not slow down protocol work.